.github/workflows/pr-build.yml

```diff
@@ -22,10 +23,16 @@ jobs:
       - name: Install Dependencies
         run: npm ci
       - name: Install Alternate @cds/core Version
-        if: ${{matrix.cds-core != ''}}
+        if: ${{matrix.cds-core == '' && matrix.angular == '13'}}
         run: npm install ${{matrix.cds-core}}
```

Comment:
This should probably be `matrix.angular`?
I'd guess that's what's leading to the current error in `PR Build / PR Build (14) `

Reply:
I've got it running now, it checks the cds-core integration in both angular versions. Not sure we need it, but it can't hurt.

Reply:
Yeah, with this check it makes sense. Thanks